apps/api/routes/documents.py

```
@@ -0,0 +1,122 @@
+            conn.close()
+
+        logger.info(
+            "Listed %d documents (total=%d, limit=%d, offset=%d)",
+            len(result.documents), result.total, limit, offset
+        )
+
+    except HTTPException:
+        # Re-raise HTTP exceptions
+        raise
+    except ValueError as e:
+        logger.exception("Validation error in list documents: %s", e)
```

Ruff (0.14.1)

Redundant exception object included in `logging.exception` call

(TRY401)